scripts/civictheme-uikit-version-manager/index.mjs

```diff
@@ -31,21 +30,24 @@ const THEME_PACKAGE_JSON = path.join(ROOT_DIR, 'web/themes/contrib/civictheme/pa
 const GITHUB_API_URL = 'https://api.github.com/repos/civictheme/uikit/branches';
 const GITHUB_COMMITS_API_URL = 'https://api.github.com/repos/civictheme/uikit/commits/';
 const GITHUB_REPO_URL = 'github:civictheme/uikit';
-const NPM_PACKAGE_NAME = '@civictheme/uikit';
+const NPM_RELEASE_PACKAGE_NAME = '@civictheme/sdc';
```

Comment:
We use `@civictheme/sdc` on releases and `@civictheme/uikit` on dev versions where we want to test a branch.